.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+1. **What type of issue would you like to report**?
+
+- [ ] Bug
+- [ ] Feature request
+- [ ] Other, please explain:
+
+2. **Check if the problem you are trying to report is already reported**.
+
+Please look for the existing issues covering the same case. First look on the [GitHub issue page](https://github.com/ckeditor/ckeditor-dev/issues)
```

Comment:
It's long and you're writing twice about each system, try to limit it to one by one, without going back to other:

> Have a look in [open issues list](https://github.com/ckeditor/ckeditor-dev/issues) using a search function. If your issue is already reported just put a +1 (👍) reaction in it.
>
> During our transition period you could also check [CKEditor's former bug tracker](https://dev.ckeditor.com), if the issue is there but not in GitHub, create a new one and put a reference link to the original Trac issue.